Packs/Phishing/Playbooks/Phishing_-_Generic_v3.yml

```diff
@@ -2766,7 +2763,8 @@ inputs:
     type using machine learning.
   playbookInputQuery:
 - key: GetOriginalEmail
-  value: {}
+  value:
+    simple: "True"
```

Comment:
We want it to be empty or "False" since current customers with no permissions will get an error otherwise.
If it's empty, they will get the default value from the "Process Email" playbook (which is "False" by default)